### PR TITLE
General UI fixes

### PR DESCRIPTION
**Pull request by Flechman**

### CODE REVIEW

File: app/src/main/res/layout/activity_display_event.xml [hunk 1]
```diff
@@ -34,6 +36,8 @@
         android:layout_marginTop="100dp"
         android:paddingLeft="4dp"
```
Comment (justagoodguy-omega) [inline app/src/main/res/layout/activity_display_event.xml:37]: Lint warning here

File: app/src/main/res/layout/activity_display_event.xml [hunk 2]
```diff
@@ -43,8 +47,9 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginTop="68dp"
-        android:textSize="32dp"
         android:text="@string/textview"
+        android:textColor="@color/col_on_secondary"
+        android:textSize="32dp"
```
Comment (justagoodguy-omega) [inline app/src/main/res/layout/activity_display_event.xml:52]: Here too

File: app/src/main/res/layout/activity_display_event.xml [hunk 3]
```diff
@@ -53,26 +58,28 @@
...
         android:layout_marginBottom="144dp"
         android:onClick="buttonStartDeleteEvent"
         android:text="@string/delete"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toEndOf="@+id/buttonDisplay" />
+        app:layout_constraintStart_toEndOf="@+id/buttonDisplay"
+        app:cornerRadius="50dp"/>
 
     <ImageButton
```
Comment (justagoodguy-omega) [inline app/src/main/res/layout/activity_display_event.xml:84]: And here

**Review by justagoodguy-omega — CHANGES_REQUESTED** [review]
A couple lint warnings (I know they were there before but now that you changed the file it's all yours :D

**Review by TheImpressiveDonut — APPROVED** [review]
There is only some small changes in xml files so I have nothing to add or comment on. Nice code !

**Review by justagoodguy-omega — APPROVED** [review]

File: app/src/main/res/layout/activity_display_event.xml [hunk 4]
```diff
@@ -52,26 +57,28 @@
...
         app:layout_constraintEnd_toStartOf="@+id/buttonDisplayDelete"
-        app:layout_constraintStart_toStartOf="parent" />
+        app:layout_constraintStart_toStartOf="parent"
+        app:cornerRadius="50dp"/>
 
     <Button
         android:id="@+id/buttonDisplayDelete"
         android:layout_width="0dp"
-        android:layout_height="85dp"
+        android:layout_height="65dp"
```
Comment (hugolan) [inline app/src/main/res/layout/activity_display_event.xml:73]: why did you change this?

**Review by hugolan — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_display_event.xml (hunk 4 above)
Comment (Flechman) [reply app/src/main/res/layout/activity_display_event.xml:73]: Because it was prettier like this

**Review by hugolan — APPROVED** [review]

**Pull request merged**